analysis_ready_planetscope/mask_classification.js

```diff
@@ -0,0 +1,38 @@
+//VERSION=3
+//cloud mask classification
+
+function setup() {
+    return {
+        input: [{
+            bands: ["cloud_mask"]
+        }],
+        output: {
+            bands: 4
+        }
+    }
+}
+
+let nodatavalue = -999
+function evaluatePixel(samples) {
```

Comment:
For `SIMPLE` mosaicking I'd suggest using `sample` instead of `samples`

Reply:
Done! Thanks!